### PR TITLE
[mobile] Move items not in guideline to Etc

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/948
[Problem] Main menu items not corresponding to the guideline
[Solution] Move additional items to Etc

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/etc/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+				</a>
+			</li>
+			<li class="ui-li-anchor">
+				<a href="graphs/index.html">
+					Graphs
+				</a>
+			</li>
+
+			<!-- @todo need to review below samples -->
+			<ul class="ui-listview ui-content-area">
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/etc/index.html:23]: Move this `ul` over `li` elements above

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

File: examples/mobile/UIComponents/components/etc/index.html (hunk 1 above)
Comment (lslachciak) [reply examples/mobile/UIComponents/components/etc/index.html:23]: done

**Comment by lslachciak** [comment]
Improved PR according to the @TomaszLukawskiSam comment

**Review by TomaszLukawskiSam — APPROVED** [review]
OK

**Pull request merged**